account_subsequence_fiscal_year/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,9 @@
+This module extends the functionality of Accounting Odoo module, to
+generate custom sub sequences for accounting entries, if fiscal years are not "classic".
+(ie: last day = 31 / last month = 12)
+
+without this module, in such cases (for example, for fiscal year from 01 April to 31 May)
```

Comment:
```suggestion
without this module, in such cases (for example, for fiscal year from 01 April 2020 to 31 May 2021)
```
You mean this? I just try to make sense. :)

Reply:
thanks.